src/store/Transaction.ts

```diff
@@ -226,6 +228,12 @@ export default {
             }
 
             state.filteredTransactions = TransactionFilterService.filter(state, currentSignerAddress);
+            if (!!multisigAddresses.length) {
+                multisigAddresses.forEach((address) => {
+                    state.filteredTransactions = [...state.filteredTransactions, ...TransactionFilterService.filter(state, address)];
+                });
+                state.filteredTransactions = TransactionFilterService.filter(state, currentSignerAddress);
```

Comment:
`state.filteredTransactions = TransactionFilterService.filter(state, currentSignerAddress);`

it called inside and outside if statement. you should refactor it.

Reply:
condition added to make sure partial transactions needs to be signed always shows up first thing on the list